### DISCUSSION TITLE
How to check compatibility

### DISCUSSION THREAD

**Discussion by pdob-git** [vuejs/language-tools#1771] [Q&A]
How can I can compatibility of particular version of Volar.
I cannot install Volar 0.40 on VsCode 1.62.3
But how can I check which VsCode version  I need?

**Comment by johnsoncodehk** [comment]
Unfortunately there is no reliable way to do it. And old version extension is not recommend to use because there are many changes.

Maybe you can try modify required client version at https://github.com/johnsoncodehk/volar/blob/2758510a8e519563f16ce023c700ab98b8b65ad8/extensions/vscode-vue-language-features/package.json#L19 to build and see what don't work, I will help if I can.
